test/extensions/filters/network/thrift_proxy/integration.cc

```diff
@@ -0,0 +1,115 @@
+#include "test/extensions/filters/network/thrift_proxy/integration.h"
+
+#include <algorithm>
+#include <fstream>
+
+#include "test/test_common/environment.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace ThriftProxy {
+
+const std::string& PayloadOptions::modeName() const {
+  static const std::string success = "success";
```

Comment:
nit: technically a violation of the static initialization fiasco rule. The easy workaround is to make these `char[]` or just inline them below, which is clearer IMHO.